docs/sql-ref-syntax-ddl-alter-view.md

```diff
@@ -19,4 +19,217 @@ license: |
+|c2                          |string                                               |null   |
+|                            |                                                     |       |
+|# Detailed Table Information|                                                     |       |
+|Database                    |tempdb1                                              |       |
+|Table                       |v2                                                   |       |
+|Table Properties            |[created.by.user=John, created.date=01-01-2001, ....]|       |
++----------------------------+-----------------------------------------------------+-------+
+
+-- Use `DESC TABLE EXTENDED tempdb1.v2` before and after the `ALTER VIEW` to verify the change.
+-- Remove the key created.by.user and created.date from TBLPROPERTIES
+ALTER VIEW tempdb1.v2 UNSET TBLPROPERTIES ('created.by.user', 'created.date');
+
+--Use `DESC TABLE EXTENDED tempdb1.v2` to verify the changes
+DESC TABLE EXTENDED tempdb1.v2;
+
++----------------------------+----------+-------+
+|col_name                    |data_type |comment|
++----------------------------+----------+-------+
+|c1                          |int       |null   |
+|c2                          |string    |null   |
+|                            |          |       |
+|# Detailed Table Information|          |       |
+|Database                    |tempdb1   |       |
+|Table                       |v2        |       |
+|Table Properties            |[....]    |       |
++----------------------------+----------+-------+
+
+-- Do the select on tempdb.view1 before and after the `ALTER VIEW` statement to verify.
+-- Change the view definition
+ALTER VIEW tempdb1.v2 AS SELECT * FROM tempdb1.v1;
+
+-- Use `DESC TABLE EXTENDED` to verify
+DESC TABLE EXTENDED tempdb1.v2;
+
++----------------------------+---------------------------+-------+
+|col_name                    |data_type                  |comment|
++----------------------------+---------------------------+-------+
+|c1                          |int                        |null   |
+|c2                          |string                     |null   |
+|                            |                           |       |
+|# Detailed Table Information|                           |       |
+|Database                    |tempdb1                    |       |
+|Table                       |v2                         |       |
+|Type                        |VIEW                       |       |
+|View Text                   |select * from tempdb1.v1   |       |
+|View Original Text          |select * from tempdb1.v1   |       |
++----------------------------+---------------------------+-------+
+{% endhighlight %}
+
+#### Related Statements
+- [describe-table](sql-ref-syntax-aux-describe-table.html)
+- [create-view](sql-ref-syntax-ddl-create-view.html)
+- [drop-view](sql-ref-syntax-ddl-drop-view.html)
+- [select](sql-ref-syntax-qry-select.md)
```

Comment:
This link isn't valid; it points to markdown. Remove it?

Reply:
removed